src/include/flow.h

```diff
@@ -156,8 +156,8 @@ class RXTracking {
  public:
   using MachnetPktHdr = net::MachnetPktHdr;
 
-  // 64-bit SACK bitmask => we can track up to 64 packets
-  static constexpr std::size_t kReassemblyMaxSeqnoDistance = 64;
+  // 256-bit SACK bitmask => we can track up to 256 packets
+  static constexpr std::size_t kReassemblyMaxSeqnoDistance = 256;
```

Comment:
Maybe at this point it is good to set the size of the reassembly queue, using the size of the relevant packet header directly. E.g. `sizeof(sack_bitmap)*sizeof(sack_bitmap_bucket)` or so.
And static_assert that it is a power of two.